build.gradle

```diff
@@ -4,37 +4,14 @@ equoIde {
 	welcome().openUrl('https://github.com/diffplug/spotless/blob/main/CONTRIBUTING.md')
 	gradleBuildship().autoImport('.')
 }
-
 repositories {
 	mavenCentral()
 }
-
-apply from: rootProject.file('gradle/java-publish.gradle')
-apply from: rootProject.file('gradle/changelog.gradle')
 allprojects {
 	apply from: rootProject.file('gradle/error-prone.gradle')
-	apply from: rootProject.file('gradle/rewrite.gradle')
 	apply from: rootProject.file('gradle/spotless.gradle')
 }
+apply from: rootProject.file('gradle/changelog.gradle')
+apply from: rootProject.file('gradle/java-publish.gradle')
+apply from: rootProject.file('gradle/rewrite.gradle')
 apply from: rootProject.file('gradle/spotless-freshmark.gradle')
```

Comment:
- before: java-publish, then changelog, then allprojects
- after: allprojects, then changelog, then java-publish

That's a risky change. I think they depend on each other, but it takes time to confirm, and we won't notice it's broken until a publish fails. Try to make your PR's more focused on a single change and they are easier to merge.